port/win/env_win.cc

```diff
@@ -954,7 +956,7 @@ class WinWritableFile : public WritableFile {
 
     DWORD bytesWritten = 0;
```

Comment:
Add ` assert(numBytes <= std::numeric_limits<DWORD>::max());` before the OS call